mapper/thin_edge_json/src/json.rs

```diff
@@ -282,12 +276,66 @@ impl ThinEdgeJsonError {
 
 mod tests {
     use super::*;
-
+    use crate::measurement::*;
     fn test_timestamp() -> DateTime<FixedOffset> {
         FixedOffset::east(5 * 3600)
             .ymd(2021, 04, 08)
             .and_hms(0, 0, 0)
     }
+    #[test]
+    fn test_str_with_invalid_time_stamp() {

```

Comment:
Please use unique spelling of `time_stamp` and `timestamp` in code:

```
fn test_str_with_invalid_time_stamp()
```

should be:

```
fn test_str_with_invalid_timestamp()
```

Reply:
done